src/pages/Account/Create.tsx

```diff
@@ -1,11 +1,10 @@
-import * as React from 'react';
-import { RouteProps } from 'react-router';
+import React from 'react';
 import ManageAccountContainer, {
   ManageAccountModes,
 } from '../../features/Account/ManageAccountContainer';
 
-const CreateAccountContainer = (props: RouteProps) => {
-  return <ManageAccountContainer mode={ManageAccountModes.CREATE} {...props} />;
+const CreateAccountPage: React.FC = () => {
```

Comment:
why are we removing props here?

Reply:
We're getting rid of withRouter HOC in favor of hooks like useHistory or useParams, so we don't have to propagate the router params all the way down the component tree